Train.json

```diff
@@ -880,6 +902,28 @@
 				{"name": "passengers", "value": 144}
 			]
 		},
+		{
+			"id": 355,
+			"group": 2,
+			"name": "DSB MG",
+			"shortcut": "IC4",
+			"speed": 200,
+			"weight": 160,
+			"force": 150,
+			"length": 86,
+			"drive": 2,
+			"reliability": 0.8,
```

Comment:
Das ist schon fast zu gut für die Karren 😂

Reply:
Und: Wo Zulassung Libyen? 🤪

(Gibt es da überhaupt Bahngleise?! OpenRailwayMap zeigt da gar nichts an...)

Reply:
Kann doch weitergehen, mit auschliseßlich Finnland fahrzeugen ^^

Reply:
Natürlich wäre das in der Praxis mit wie das hier laufen würde möglich. Aber in der Theorie sind das 2 nicht zusammenhängende Netze. Und das mag TC nicht, das habe ich schon herausgefunden^^

Reply:
Dann machen wir ne Strecke dahin mit vmax 0.5 oder so mit zulöassung US xD

Reply:
> Natürlich wäre das in der Praxis mit wie das hier laufen würde möglich. Aber in der Theorie sind das 2 nicht zusammenhängende Netze. Und das mag TC nicht, das habe ich schon herausgefunden^^

Dem Spiel dürfte das glaube ich eher egal sein 😅

Es ist nur die Überprüfung auf mögliche Fehler, die dabei Alarm schlägt. Aber das lässt sich auch schnell eben anpassen, dass zwei Netze in Ordnung und gewollt sind :D

Reply:
Erstmal muss Schweden weiter/fertig gemacht werden^^